spec/Makefile

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements…
+# (your license header…)
+
+PYTHON  := python3
```

Comment:
Which files does this generate for the spec? Will the server implementation can use the generated spec file? 
https://github.com/apache/polaris/tree/main/spec#generated-specification-files

Reply:
The current functionality is only for linting the spec yaml, and not for generating any models.

Reply:
okay, generating the spec would be better IMO it's easier to parse those requests like proto instead of breaking changes?

Reply:
I have added some functionality to generate java models from the spec.